model/src/main/java/org/openremote/model/webhook/Webhook.java

```diff
@@ -0,0 +1,111 @@
+package org.openremote.model.webhook;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Webhook {
+
+    public enum Method {
```

Comment:
This enum already exists refer to `HTTPAgentLink` and you'll see `HTTPMethod` being used